x11-packages/alacritty/build.sh

```diff
@@ -3,98 +3,79 @@ TERMUX_PKG_DESCRIPTION="A fast, cross-platform, OpenGL terminal emulator"
 TERMUX_PKG_LICENSE="Apache-2.0, MIT"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
 TERMUX_PKG_VERSION="0.15.1"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=3
```

Comment:
The revision should only be incremented when a change to the package is actually merged.
So this just needs to be a `2`.
```suggestion
TERMUX_PKG_REVISION=2
```